jumanji/environments/logic/game_2048/env_test.py

```diff
@@ -61,7 +61,7 @@ def test_game_2048__step_jit(game_2048: Game2048) -> None:
     """Confirm that the step is only compiled once when jitted."""
```

Comment:
Nice efforts in adding additional tests :raised_hands: , however, I think no need to add these tests since we checked the action masking `test_game_2048__get_action_mask`. I don't if @clement-bonnet  do you think we need to fix these tests and add them (maybe we can keep `test_game_2048__step_action_mask `) ?